test/lit.site.cfg.py.in

```diff
@@ -2,6 +2,11 @@
 
 import sys
 
+ON = True
```

Comment:
use

```
def to_bool(value):
    return value in [True, 'TRUE', 'ON', 1]
```

rather than global constants

Reply:
yaml vibes here

Reply:
The province of Ontario is true :trollface:

Reply:
Not to be confused with the province of Offtario, which is false